recipe/meta.yaml

```diff
@@ -7,16 +7,16 @@ package:
 
 source:
   url: https://pypi.io/packages/source/d/distributed/distributed-{{ version }}.tar.gz
-  sha256: e3d8053720cd58a3a254e8862900fec1e0940e9e34b044482142d836cd1ed892
+  sha256: e59bb9061bbe496b017c4374d4e76bed0e22df22bbdbd93c1cc5d60642c20671
 
 build:
   number: 0
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
   entry_points:
-    - dask-scheduler = distributed.cli.dask_scheduler:go
-    - dask-ssh = distributed.cli.dask_ssh:go
-    - dask-worker = distributed.cli.dask_worker:go
+    - dask-scheduler = distributed.cli.dask_scheduler:main
+    - dask-ssh = distributed.cli.dask_ssh:main
+    - dask-worker = distributed.cli.dask_worker:main
```

Comment:
As the `entry_points` changed upstream in response to the `click` `8.1.0` release ( https://github.com/dask/distributed/pull/6018 ), needed to make similar changes here